app.js

```diff
@@ -5,10 +5,11 @@ const { prependHttps } = require("./helpers");
 
 async function createBasicCluster(action, settings) {
   const {
-    name, locationType, region, zone, controlPlaneReleaseChannel, version, numberOfNodes,
+    name, locationType, zone, controlPlaneReleaseChannel, version, numberOfNodes,
     enableAutoscaling, minNode, maxNode, machineType, nodeImage, diskType,
     diskSize, diskEncryptionKey, waitForOperation, network, subnetwork,
   } = action.params;
+  const region = action.params.region || settings.region;
```

Comment:
If making a zonal cluster, Region is not required. There are many combination of impossible cluster that can be configured and GCP does a great job of catching them all and providing a useful error message. We could enhance validation, but it would be more like ensuring region is defined when cluster type is regional. That's only necessary in one method that I can't make work because of a bug in the autocomplete, which I'll be backlogging soon.